pkg/alertmanager/alertmanager.go

```diff
@@ -188,19 +188,25 @@ func New(cfg *Config, reg *prometheus.Registry) (*Alertmanager, error) {
 	am.state = newReplicatedStates(cfg.UserID, cfg.ReplicationFactor, cfg.Replicator, cfg.Store, am.logger, am.registry)
 	am.persister = newStatePersister(cfg.PersisterConfig, cfg.UserID, am.state, cfg.Store, am.logger, am.registry)
 
-	am.wg.Add(1)
```

Comment:
Note to reviewers: moved below because this was used to track the maintenance goroutine which we're not responsible to start.